app/assets/javascripts/govuk-component/option-select.js

```diff
@@ -62,20 +81,19 @@
     return checkedString;
   };
 
-  OptionSelect.prototype.attachOpenCloseToggleIndicator = function attachOpenCloseToggleIndicator(){
-    this.$optionSelect.find('.js-container-head').append('<div class="js-toggle-indicator"></div>');
-  };
 
-  OptionSelect.prototype.toggleOptionSelect = function toggleOptionSelect(){
+  OptionSelect.prototype.toggleOptionSelect = function toggleOptionSelect(e){
     if (this.isClosed()) {
       this.open();
     } else {
       this.close();
     }
+    e.preventDefault();
   };
 
   OptionSelect.prototype.open = function open(){
     if (this.isClosed()) {
+      this.$optionSelect.find('button').attr('aria-expanded', true);
```

Comment:
I would be tempted to call `find('.js-container-head')` rather than just `button` so it is slightly clearer as to what element you are looking for. But the way you have done it also works and is fine.